examples/reader.rs

```diff
@@ -0,0 +1,27 @@
+use std::fs::File;
+
+use pdfium_render::prelude::*;
+
+fn how_many_pages(pdfium: &Pdfium, path: &str) -> Result<u16, PdfiumError> {
+    let reader = File::open(path).map_err(PdfiumError::IoError)?;
+    let document = pdfium.load_pdf_from_reader(reader, None)?;
+
+    Ok(document.pages().len())
+}
+
+pub fn main() -> Result<(), PdfiumError> {
```

Comment:
this example is a bit silly - i.e. might not be worth including

this was mostly to convince myself my changes made a difference by repro-ing a toy example of how I want to use this codepath